frontend/packages/console-plugin-shared/OWNERS

```diff
@@ -0,0 +1,8 @@
+reviewers:
+  - mareklibra
+  - rawagner
+approvers:
+  - mareklibra
+  - rawagner
```

Comment:
I think reviewers/approvers should be the same as https://github.com/openshift/console/blob/master/frontend/packages/console-dynamic-plugin-sdk/OWNERS and later we can expand